src/read/archive.rs

```diff
@@ -93,6 +90,56 @@ impl<'data, R: ReadRef<'data>> ArchiveFile<'data, R> {
                     file.symbols = (symtbl, len);
                 }
             }
+
+            // Big archive member table lists file entries with offsets and names. To avoid
+            // potential infinite loop (members are double-linked list), the iterator goes
+            // through the table instead of real members.
+            let mut member_table_offset = parse_u64_digits(&file_header.memoff, 10)
+                .read_error("Invalid offset for member table of AIX big archive")?;
+            if member_table_offset > 0 {
+                // The table itself is also a file with header.
+                let header = data
+                    .read::<archive::AixHeader>(&mut member_table_offset)
+                    .read_error("Invalid AIX big archive member header")?;
+                let terminator = data
+                    .read_bytes(&mut member_table_offset, 2)
+                    .read_error("Invalid AIX big archive terminator")?;
+                if terminator != archive::TERMINATOR {
+                    return Err(Error("Invalid AIX big archive terminator"));
+                }
+
+                // Structure of member table:
+                // Number of entries (20 bytes)
+                // Offsets of each entry (20*N bytes)
+                // Names string table (the rest of bytes to fill size defined in header)
+                let table_size = parse_u64_digits(&header.size, 10)
+                    .read_error("Invalid AIX big archive member size")?;
+                let members_count = data
+                    .read_bytes_at(member_table_offset, 30)
```

Comment:
Should this have been a length of 20?

I'll do a followup PR for some other things, so I'll change this if needed.

Reply:
Ah, it should be 20, Thanks!